mbta-performance/chalicelib/lamp/ingest.py

```diff
@@ -202,6 +169,34 @@ def _recalculate_fields_from_gtfs(pq_df: pd.DataFrame, service_date: date):
     return pq_df[S3_COLUMNS]
 
 
+def _average_scheduled_headways(pq_df: pd.DataFrame, service_date: date) -> pd.DataFrame:
+    """Bucket scheduled headways into 30 minute buckets.
+
+    We do this so as to smooth the benchmark headways for the data dashboard.
+    TODO: do this with branch headways as well
```

Comment:
I think what we're calculating right now is branch headways? At least on the green line?

Reply:
Will do this as a follow up, https://github.com/transitmatters/mbta-performance/issues/25